bndtools.core.test/build.gradle

```diff
@@ -0,0 +1,37 @@
+import aQute.bnd.gradle.TestOSGi
+import org.apache.tools.ant.taskdefs.condition.Os
+
+task testosgi(type: TestOSGi) {
```

Comment:
There is already a `testOSGi` task defined by the Bnd Gradle plugin. Why are you not just configuring that task? And also you need to use the configuration avoidance style of writing gradle task for this build.

Reply:
Im not a Gradle expert, so please excuse my ignorance, but what's the "configuration avoidance " style? I knew there was already a testOSGi task but I didn't know how to reconfigure it.

Reply:
See https://github.com/bndtools/bnd/blob/35661c2dd92f56993c78cf6c984323af0f41d42f/biz.aQute.bnd.gradle/build.gradle#L22-L53 for some examples of configuring tasks that avoid executing the configuration when the task is not part of the task graph.

Reply:
Fixed, thanks for the pointer.